python/lsst/ts/mtdometrajectory/dome_trajectory.py

```diff
@@ -429,9 +435,9 @@ async def report_vignetted_loop(self):
                 )
                 await asyncio.sleep(VIGNETTING_MONITOR_INTERVAL)
         except asyncio.CancelledError:
-            self.log.info("report_vignetted_loop ends")
+            self.log.info("report_vignetted_loop ends.")
         except Exception as e:
-            self.log.exception(f"report_vignetted_loop failed: {e!r}")
+            self.log.exception(f"report_vignetted_loop failed: {e!r}.")
```

Comment:
I think you could remove the exception from this part because `self.log.exeception` will also log the exception. I'd just replace this with 

```
except Exception:
    self.log.exception(f"report_vignetted_loop failed.")
```